src/librustdoc/clean/types.rs

```diff
@@ -754,7 +754,17 @@ impl Item {
             .other_attrs
             .iter()
             .filter_map(|attr| {
-                if is_json {
+                // target_feature is special-cased because cargo-semver-checks uses it
+                if let hir::Attribute::Parsed(AttributeKind::TargetFeature(features, _)) = attr {
+                    let mut output = String::new();
+                    for (i, (feature, _)) in features.iter().enumerate() {
+                        if i != 0 {
+                            output.push(',')
+                        }
+                        output.push_str(feature.as_str());
+                    }
+                    Some(format!("#[target_feature(enable=\"{output}\")]"))
```

Comment:
is this right? shouldn't it be multiple enable = "..." strings concatenated?

Reply:
Yeah this is allowed lmao, also surprised me